requirement.txt

```diff
@@ -1,2 +1,4 @@
 ostorlab
-rich
\ No newline at end of file
+rich
```

Comment:
```suggestion
```

Reply:
Isn't this part of ostorlab ?

Reply:
But explicity used separately. If we move out, we need to keep it.